docs/products/introduction.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_label: 'Introduction'
+sidebar_position: 1
+---
+
+# Introduction
+
+ZKPool is an open and permissionless platform that connects various ZKP projects and provers, offering multiple capabilities:
+
+- ZKPool gathers the capacity of smaller provers and acts as a super prover proxy, providing necessary stake to ZKP projects.
+- ZKPool adopts a strategy to maximize rewards and minimize the risks of being slashed.
```

Comment:
"adopts a strategy" what does it refer to?

Reply:
They include our infrastructure and price strategy.